server/src/controllers/christmas-tree.es6

```diff
@@ -441,4 +445,56 @@ christmasTree.cancelOne = (req, res) => {
       res.status(404).send();
     });
 };
+
+christmasTree.getPermits = (req, res) => {
+  treesDb.christmasTreesPermits
+    .findAll({
+      attributes: ['forestId', 'paygovTrackingId', 'updatedAt', 'quantity', 'totalCost'],
+      where: {
+        forestId: req.params.forestId,
+        status: 'Completed',
+        updatedAt: {
+          [Op.gte]: req.params.startDate,
+          [Op.lte]: req.params.endDate
+        }
+      },
+      order: [['updatedAt', 'ASC']]
+    })
+    .then(results => {
+      if (results) {
+        let permits = [];
```

Comment:
let's pull out the summary into an additional function L464-476.. that is called within the getAll()

Reply:
task on 285